api/PannerNode.json

```diff
@@ -61,7 +61,7 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": "≤79"
+              "version_added": "79"
```

Comment:
This is the `PannerNode` constructor. The Safari data is null here, and that's OK, but I'll just note that `webkitAudioPannerNode exists in Safari. It doesn't work as a constructor, though.

As a follow-up, it would be good to add `"alternative_name": "webkitAudioPannerNode"` for Safari. I don't know why just that interface is exposed, when many others are not, but that's the situation.